cdk/bin/cdk.ts

```diff
@@ -1,47 +1,65 @@
 import 'source-map-support/register';
-import { WiresFeeds } from '../lib/wires-feeds';
-import { Newswires } from '../lib/newswires';
-import { App } from 'aws-cdk-lib';
 import { RiffRaffYamlFile } from '@guardian/cdk/lib/riff-raff-yaml-file';
+import { App } from 'aws-cdk-lib';
+import {
+	PollerId,
+	pollerIdToLambdaAppName,
+	POLLERS_CONFIG,
+} from '../../shared/pollers';
+import { Newswires } from '../lib/newswires';
+import { WiresFeeds } from '../lib/wires-feeds';
+import { STACK } from '../../shared/constants';
 
 const app = new App();
 
-// FIXME we should probably have a dedicated stack for the Newswires app for e.g. cost explorer purposes (as the App tag needs to be different for riff-raff etc. to differentiate)
-const stack = 'editorial-feeds';
-
 const env = {
 	region: 'eu-west-1',
 };
 
 const codeWiresFeeds = new WiresFeeds(app, 'WiresFeeds-CODE', {
 	env,
-	stack,
+	stack: STACK,
 	stage: 'CODE',
 });
 
 const prodWiresFeeds = new WiresFeeds(app, 'WiresFeeds-PROD', {
 	env,
-	stack,
+	stack: STACK,
 	stage: 'PROD',
 });
 
 new Newswires(app, 'Newswires-CODE', {
 	env,
-	stack,
+	stack: STACK,
 	stage: 'CODE',
 	domainName: 'newswires.code.dev-gutools.co.uk',
 	enableMonitoring: false,
+	sourceQueue: codeWiresFeeds.sourceQueue,
 	fingerpostQueue: codeWiresFeeds.fingerpostQueue,
 }).addDependency(codeWiresFeeds);
 
 new Newswires(app, 'Newswires-PROD', {
 	env,
-	stack,
+	stack: STACK,
 	stage: 'PROD',
 	domainName: 'newswires.gutools.co.uk',
 	enableMonitoring: false,
+	sourceQueue: prodWiresFeeds.sourceQueue,
```

Comment:
is there a particular reason for creating the source queue as part of a different stack?

Reply:
I think because FIP have a reference to it, so I think we wanted to decouple, such that we could theoretically tear down the newswires stack without impacting FIP writing to the queue. @andrew-nowak could probably confirm...